solr/contrib/ltr/src/java/org/apache/solr/ltr/SolrQueryRequestContextUtils.java

```diff
@@ -47,12 +47,12 @@ public static FeatureLogger getFeatureLogger(SolrQueryRequest req) {
 
   /** scoring query accessors **/
 
-  public static void setScoringQuery(SolrQueryRequest req, LTRScoringQuery scoringQuery) {
-    req.getContext().put(SCORING_QUERY, scoringQuery);
+  public static void setScoringQuery(SolrQueryRequest req, LTRScoringQuery[] scoringQueries) {
```

Comment:
3/n `getScoringQuery` already is renamed to `getScoringQueries` to reflect the changed signature, how about renaming `setScoringQuery` to `setScoringQueries` also?  https://github.com/cpoerschke/lucene-solr/commit/2579aee9f2e76321a08754d89630f635ae6899b7 does that plus `SCORING_QUERY` to `SCORING_QUERIES` too.

Reply:
absolutely reasonable!